.github/workflows/IntegrationTest.yml

```diff
@@ -0,0 +1,27 @@
+name: Spark Connector Integration Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: sbt integtation test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Decrypt snowflake.json for testing
+      run: ./.github/scripts/decrypt_secret.sh
+      env:
+        SNOWFLAKE_TEST_PROFILE_SECRET: ${{ secrets.SNOWFLAKE_TEST_PROFILE_SECRET }}
+    - name: Run tests
+      run: |
+          sbt compile
+          sbt test
+          sbt it:test
```

Comment:
_sbt it: test_ includes _sbt compile_ and _sbt test_ . don't need call them two times